src/base/base.ts

```diff
@@ -79,8 +79,10 @@ export default class Base {
    * @returns {Promise<GetResponse>}
    */
   public async get(key: string): Promise<GetResponse> {
-    const trimmedKey = key.trim();
-    if (!trimmedKey.length) {
+    const trimmedKey = key?.trim();
+    if (!trimmedKey) {
+      throw new Error('Key is empty');
+    } else {
       throw new Error('Key is empty');
     }
```

Comment:
Remove the else block we don't need those anymore

Reply:
Ah yeah you're right

Reply:
Fixed